commands.json

```diff
@@ -4163,6 +4194,37 @@
     "since": "5.0.0",
     "group": "sorted_set"
   },
+  "ZRANDMEMBER": {
+    "summary": "Get one or multiple random elements from a sorted set",
+    "complexity": "O(N*log(N)) where N is the number of elements returned",
```

Comment:
why is this different than both HRANDFIELD and SRANDMBMER?
i don't see any extra work we do here for zsets that we don't do for hashes.

Reply:
Haven't realized it wasn't log(N).

Resolved via ca2bd9a3da2ad5e88dd2ef434cf21e6062b8649f